Update README-Portuguese.md (#2116)
the-turing-way/the-turing-way: README-translated/README-Portuguese.md

O projeto é coordenado pelos investigadores principais **Kirstie Whitaker** (fundadora) e **Malvika Sharan** (desenvolvedor de comunidade) e hospedado em [The Alan Turing Institute](https://www.turing.ac.uk/).

Você pode ler [_The Turing Way_ - Processo de reconhecimento](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) e [Registro de contribuições](https://the-turing-way.netlify.app/afterword/contributors-record.html) para saber como reconhecemos seu trabalho e como nossos colaboradores são destacados no projeto.
Por favor veja a [tabela de contribuidores](#contribuidores) com o perfil no Github de todos nossos contribuidores.

### Como contribuir

:construction: Este repositório está em constante desenvolvimento e **todos** são incentivados a nos ajudar a construir algo que seja útil para muitos. :construction:

Esperamos que todos que participarem do nosso projeto sigam nosso [código de conduta](CODE_OF_CONDUCT.md) e leia nosso [guia de contribuições](CONTRIBUTING.md) para mais informações sobre como começar.
Queremos encontrar nossos colaboradores onde eles estão.

> Portanto, fornecemos vários pontos de entrada para você contribuir com base em seu interesse, disponibilidade ou requisitos de habilidade.

Reviewer: ```suggestion
Por isso, fornecemos vários pontos de entrada para você contribuir com base em seu interesse, disponibilidade ou habilidades.
```

> ![Essa imagem mostra seis tipos de contribuições que todos podem fazer. São: desenvolver e compartilhar, manter e melhorar, compartilhar recursos, revisar e atualizar, torná-lo global por meio da tradução e compartilhar as melhores práticas](book/website/figures/README_imgs/README_contributions.png)

Reviewer: ```suggestion
![Esta imagem mostra seis tipos de contribuições que todos podem fazer. São elas: desenvolver e compartilhar; manter e melhorar; compartilhar recursos; revisar e atualizar; globalizar por meio da tradução; e compartilhar as melhores práticas](book/website/figures/README_imgs/README_contributions.png)
```

> *As contribuições incluem desenvolvimento e compartilhamento de novos capítulos; manutenção e melhoria dos capítulos existentes; compartilhamento de recursos do _The Turing Way_; revisão e atualização de materiais desenvolvidos anteriormente; tradução do seu capítulo para ajudar a tornar este projeto globalmente acessível e compartilhar as melhores práticas de pesquisa.*

Reviewer: ```suggestion
*As contribuições incluem desenvolvimento e compartilhamento de novos capítulos; manutenção e melhoria dos capítulos existentes; compartilhamento de recursos do _The Turing Way_; revisão e atualização de materiais desenvolvidos anteriormente; tradução de capítulos para ajudar a tornar este projeto globalmente acessível; e compartilhar as melhores práticas de pesquisa.*
```

> Os membros da comunidade têm oportunidades de aprender novas habilidades, compartilhar suas ideias e colaborar com outros.

Reviewer: ```suggestion
Pessoas membras da comunidade têm oportunidades de aprender novas habilidades, compartilhar suas ideias e colaborar com outras.
```

> Eles também recebem oportunidades de mentoria no projeto à medida que fazem contribuições para _The Turing Way_ ou outros projetos de código aberto e são incentivados a serem mentores de novos participantes do projeto.

Reviewer: ```suggestion
Elas também recebem oportunidades de mentoria no projeto à medida que fazem contribuições para _The Turing Way_ ou outros projetos de código aberto e são incentivadas a serem mentoras de novas pessoas que participem do projeto.
```

> Nós criamos um [pacote promocional](https://github.com/alan-turing-institute/the-turing-way/tree/master/communications/promotion-pack) para ajudá-lo a apesentar e compartilhar sobre _The Turing Way_ na sua rede.

Reviewer: ```suggestion
Nós criamos um [pacote promocional](https://github.com/alan-turing-institute/the-turing-way/tree/master/communications/promotion-pack) para te ajudar a apesentar e compartilhar _The Turing Way_ nas suas redes.
```

> Se você possuir alguma dúvida, [entre em contato](#entre-em-contato).

Reviewer: ```suggestion
Se você tiver alguma dúvida, [entre em contato](#entre-em-contato).
```

### Citando as ilustrações do _The Turing Way_
<p align="center">
  <img src="book/website/figures/evolution-open-research.jpg" alt="Este é um exemplo de uma das ilustrações de The Turing Way. Tenta mostrar a evolução para uma era de ciência aberta" width="600">
</p>

> As ilustrações do _The Turing Way_ são criadas por artistas da [Scriberia](https://www.scriberia.co.uk/) como parte do [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash) em Manchester em 17 de maio de 2019 e em Londres em 28 de maio de 2019 e 21 de fevereiro de 2020.  

Reviewer: ```suggestion
As ilustrações do _The Turing Way_ são criadas por artistas da [Scriberia](https://www.scriberia.co.uk/) como parte dos eventos [_The Turing Way_ book dashes](https://github.com/alan-turing-institute/the-turing-way/tree/main/workshops/book-dash).  
```

> Eles descrevem uma variedade de conteúdo do manual, esforços colaborativos na comunidade e o projeto _The Turing Way_ em geral.  

Reviewer: ```suggestion
Elas descrevem vários conteúdos do livro, os esforços colaborativos da comunidade e o projeto _The Turing Way_ em geral.
```

### Entre em contato

#### Email

Você pode entrar em contato com a equipe *The Turing Way* enviando um e-mail para [theturingway@gmail.com](mailto:theturingway@gmail.com).

Você também pode entrar em contato com **Malvika Sharan** pelo e-mail [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk) ou **Kirstie Whitaker** pelo e-mail [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).

#### Chat

> Conecte-se com outros e discuta suas ideias no Slack usando [esse link de convite](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw).

Reviewer: ```suggestion
Conecte-se com outras pessoas e discuta suas ideias no Slack usando [este link de convite](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw).
```

> Nós também temos uma sala de chat no Gitter (se você prefere outra alternativa de código aberto para chat) e nós adoraríamos que você passasse para dizer "oi" em <https://gitter.im/alan-turing-institute/the-turing-way>.

Reviewer: ```suggestion
Nós também temos uma sala de chat no Gitter (se você prefere outra alternativa de código aberto para conversar) e adoraríamos que você passasse para dizer "oi" em <https://gitter.im/alan-turing-institute/the-turing-way>.
```
